### PR TITLE
Update UG to include view event and home command

### PR DESCRIPTION
**Pull request by tjch-o**
Now the command summary is grouped by contact, then tag, then event. Each command is sorted lexicographically within the group.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -240,6 +248,12 @@ Clears all entries from JobFestGo.
 
 Format: `clear`
 
+### Returning to home page : `home`
+
+Returns to the home page of JobFestGo regardless of what screen the user is currently on.
+
+Format: `clear`
```
Comment (rionshocker) [inline docs/UserGuide.md:255]:
```suggestion
Format: `home`
```

Comment (rionshocker) [reply docs/UserGuide.md:255]: Typo over here

**Review by rionshocker — CHANGES_REQUESTED** [review]
LGTM! Just a minor typo.

File: docs/UserGuide.md (hunk 1 above)
Comment (tjch-o) [reply docs/UserGuide.md:255]: oops my mistake will fix it rn

**Review by rionshocker — APPROVED** [review]
LGTM!

**Pull request merged**